package.json

```diff
@@ -25,13 +25,14 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react": ">=15.4.1",
-    "react-dom": ">=15.4.1"
+    "react": ">=15.5.4",
```

Comment:
I rolled back the peer dependency change and checked the minimum version for prop-types is react 15.3.0